Epsilon.Abstractions/Model/CoursePage.cs

```diff
@@ -0,0 +1,20 @@
+namespace Epsilon.Abstractions.Model;
+
+public class CoursePage
+{
+    public string Title { get; set; }
+    public string CreatedAt { get; set; }
+    public string Url { get; set; }
+    public string EditingRoles { get; set; }
+    public string PageId { get; set; }
+    public string LastEditedBy { get; set; }
+    public string Published { get; set; }
+    public string HideFromStudents { get; set; }
+    public string FrontPage { get; set; }
+    public string HTMLUrl { get; set; }
```

Comment:
Please use Html instead of HTML to correspond to naming conventions

Reply:
changed